### DISCUSSION TITLE
my website is down

### DISCUSSION THREAD

**Discussion by Parshva1** [vercel/community#5150] [Help]
### Summary

At first my website was perfectly working but after that when i updated the cloudflare dns to my domain provider then i reconfigured my vercel dns records and there seems to be no error even then i can't open my site pls help me

### Steps to Reproduce

my website was perfectly working but after that when i updated the cloudflare dns to my domain provider then i reconfigured my vercel dns records and there seems to be no error even then i can't open my site

**Comment by b0c0de** [comment]
@Parshva1 

Website not working after updating Cloudflare and Vercel DNS? Don't worry! Here's the quick fix:

**Wait**: DNS updates can take up to 24 hours to spread, so give it some time.
**Check Cloudflare**: Make sure your settings like SSL and caching are compatible with Vercel. Try disabling Cloudflare temporarily to see if it's the culprit.
**Double-check Vercel**: Confirm your DNS records are correct, no typos!
**Clear cache**: Sometimes your browser holds outdated info, try clearing it or using incognito mode.
**Server check**: Contact your web host to rule out server issues.
If none of these work, share screenshots of your settings and error messages for more specific help!

**Comment by amyegan** [comment]
Hi @Parshva1. We have some documentation about [using Cloudflare as your DNS provider](https://vercel.com/docs/integrations/cloudflare#using-cloudflare-as-your-dns-provider) that should help you. There is also some useful information in the docs for [Adding & Configuring a Custom Domain](https://vercel.com/docs/projects/domains/add-a-domain#) and [Troubleshooting domains](https://vercel.com/docs/projects/domains/troubleshooting#).

Please give that a try an let me know if you still have trouble with your domain. 🙏
